packages/design-system-docs/src/pages/startup/components.md

```diff
@@ -0,0 +1,88 @@
+
+The examples below assume you've [installed the design system using NPM]({{root}}/startup/installation/) and have already setup your build system for React.
+
+<h3>Named imports</h3>
+
+Components can be imported from the package entry point using the syntax below.
+
+<div class="ds-c-alert ds-c-alert--warn ds-u-margin-bottom--2 ds-u-font-size--small">
+  <div class="ds-c-alert__body">
+    <h3 class="ds-c-alert__heading ds-u-font-size--base">Performance note</h3>
+    <p class="ds-c-alert__text">
+      This approach may result in a larger file size than intended. Depending on what module bundler you use, all of the design system's React components may be included in the bundled file even if you didn't specifically import them. This can be avoided by using [tree shaking in Webpack](https://webpack.js.org/guides/tree-shaking/).
+    </p>
+  </div>
+</div>
+
+```jsx
+import { Button } from '@cmsgov/design-system';
+import { TextField, ChoiceList } from '@cmsgov/design-system';
+```
+
+The `@cmsgov/design-system` package is configured to automatically work with Webpack 4's tree shaking. We provide a separate entry point (`index.es.js`) and component directory (`esnext`) for ES modules to support these optimizations.
+
+<h3>Individual imports</h3>
+
+Components can also be imported from their individual export file. This method carries no risk of including unused or extra code, as you are only importing from a specific file.
+
+```jsx
+import Button from '@cmsgov/design-system/dist/components/Button/Button';
+```
+
+<h3>Usage</h3>
+
+After importing the React component, tell React to render the component in the DOM:
+
+```jsx
+ReactDOM.render(
+  <Button
+    variation="primary"
+    className="custom-btn"
+    onClick={() => alert('Button clicked!')}>
+    Button react component
+  </Button>
+  document.querySelector('#app'),
+);
+```
+
+See [React component documentation]({{root}}/components/button/#components.button.react) for more react examples, guidance, and detailed prop documentation.
+
+<h2>Using CSS components</h2>
+
+If your project doesn't use React, we provide a CSS-only version of our components. This includes styles for every component in the library, but you’ll be responsible for writing the correct markup and updating classes and DOM attributes in response to user events.
+
+This example assumes you've already added [design system styles via CSS or Sass]({{root}}/startup/sass-and-css/) by including either the CSS or Sass.
+
+```html
+<div class="ds-c-alert ds-c-alert--warn">
+  <div class="ds-c-alert__body">
+    <h3 class="ds-c-alert__heading">Warning status</h3>
+    <p class="ds-c-alert__text">
+      Alert warning body text
+    </p>
+  </div>
+</div>
+```
+
+See [component documentation]({{root}}/components/alert/) for more HTML/CSS examples and guidance.
+
+<h2 id="examples" class="ds-h2 ds-u-color--primary-darker">Examples</h2>
```

Comment:
I think if we just use <h2>Examples</h2> here that Markdown with automatically create in ID

Reply:
This was existing code. It seems like "Examples" and "Need help or ran into an issue?" all seem to have that additional class. Maybe it's best to leave it here and then address it on all the pages later on?

Reply:
Sounds good to me